src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -310,9 +310,14 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
 
             $this->loading[$id] = true;
 
+            unset($e);
             try {
                 $service = $this->$method();
             } catch (\Exception $e) {
+            } catch (\Throwable $e) {
+            }
+
+            if (isset($e)) {
```

Comment:
actually, you don't need the ``if ($e instanceof InactiveScopeException`` in the second catch block